rpc-client-api/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+#![allow(clippy::integer_arithmetic)]
+
+pub mod client_error;
+pub mod rpc_config;
+pub mod rpc_custom_error;
+pub mod rpc_deprecated_config;
+pub mod rpc_error_object;
+pub mod rpc_filter;
+pub mod rpc_request;
+pub mod rpc_response;
```

Comment:
We could drop `rpc_` from these modules? That prefix is now redundant.

Dropping `Rpc` from the types is probably way too much churn though

Reply:
Done